test/api_tictac_web/controllers/game_controller_test.exs

```diff
@@ -0,0 +1,88 @@
+defmodule Api.ApiTictacWeb.GameControllerTest do
+  use ApiTictacWeb.ConnCase
+
+  test "new_game/2 returns an empty board, game status in progess and no winner", %{conn: conn} do
+    response =
+      conn
+      |> get(game_path(conn, :new_game))
+      |> json_response(200)
+
+    expect = %{
+      "Game Status" => "In Progress",
+      "Winner" => nil,
+      "Board" => ["empty", "empty", "empty", "empty", "empty", "empty", "empty", "empty", "empty"]
```

Comment:
Are you returning an array for the board? Doesn't the story specify a string of values? Remember that if you agreed to a string of values in the story, you need to explicitly ask the client if it is ok to change it. Otherwise, they can (rightfully) claim that you did not deliver what you agreed to.